test/integration/teacher-registration/teacher_registration_utils.js

```diff
@@ -0,0 +1,97 @@
+module.exports.constants = {
+    'nextStepXpath': '//button[span[contains(text(), "Next Step")]]',
+    'generalErrorMessageXpath': '//span[@class="help-block validation-message"]/span[contains(text(),'
```

Comment:
@cwillaim we had discussed taking the `//` out in order to generalize this, is that right?